### ISSUE TITLE
Odd error string in chai-prefer-contains-to-index-of

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [microsoft/tslint-microsoft-contrib#659]
### Bug Report

-   **`tslint-microsoft-contrib` version**: 6.0.0
-   **TSLint version**: 5.11.0
-   **TypeScript version**: 3.1.2
-   **Running TSLint via**: CLI

#### TypeScript code being linted

```tsx
expect(targetUrl.indexOf(twitterAuthUrl)).to.equal(-1,'...');
```

with `tslint.json` configuration:

```json
{
    "extends": "tslint-microsoft-contrib",
    // ...
    "rules": {
        "chai-prefer-contains-to-index-of": true
    }
}
```

#### Actual behavior

`Found chai call with indexOf that can be converted to .contain assertion: `

#### Expected behavior

`Found chai call with indexOf that can be converted to .contain assertion`

Seems weird to have a `:` there. Might be a holdover from legacy formatting preferences on TSLint.

**Comment by JoshuaKGoldberg** [comment]
I'll take this on as a side mission to #489.

**Comment by JoshuaKGoldberg** [comment]
☠️  *It's time!* ☠️ 

Per #876, this repository is no longer accepting feature pull requests. TSLint is being deprecated and we recommend you switch to https://typescript-eslint.io.

Thanks for open sourcing with us, everyone!
